inc/extras.php

```diff
@@ -35,8 +35,7 @@ function twentyseventeen_body_classes( $classes ) {
 	}
 
 	// Add class if no custom header or featured images
-	$header_image = get_header_image();
-	if ( '' == $header_image && ( ! has_post_thumbnail( $post->ID ) || is_home() ) ) {
+	if ( false === has_header_image() && ( ! has_post_thumbnail() || is_home() ) ) {
```

Comment:
@grappler This could be done like this 

```php
if ( ! has_header_image() && ( ! has_post_thumbnail() || is_home() ) ) {
```
just to be consistent with the next check what do you think?